Fix:

error: implicitly declaring library function 'strlen' with type 'unsigned long (const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'sprintf' with type 'int (char *, const char *, ...)' [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2008-02-16 15:13:13.000000000 -0600
+++ configure	2024-07-24 01:49:25.000000000 -0500
@@ -20284,8 +20284,10 @@
 cat >>conftest.$ac_ext <<_ACEOF
 /* end confdefs.h.  */
 
+#include <stdio.h>
+#include <string.h>
 int
-main ()
+main (void)
 {
 char *s1 = "foo", *s2 = "bar";
          char s3[strlen(s1) + strlen(s2) + 1];
@@ -20704,8 +20704,9 @@
       #include<sys/types.h>
       #include<sys/un.h>
 
+#include <string.h>
 int
-main ()
+main (void)
 {
 struct sockaddr_un su; int i = SUN_LEN(&su);
   ;
